velox/common/file/FileSystems.h

```diff
@@ -43,6 +43,11 @@ struct FileOptions {
 
   std::unordered_map<std::string, std::string> values;
   memory::MemoryPool* pool{nullptr};
+  std::optional<int64_t> fileSize{std::nullopt};
+
+  std::optional<int64_t> getFileSize() const {
```

Comment:
What's the point of adding this getter if 'fileSize' is public?

Reply:
Removed, this was earlier returning the optional's value.